css-in-javascript/README.md

```diff
@@ -430,3 +433,185 @@
 ---
 
 CSS puns adapted from [Saijo George](https://saijogeorge.com/css-puns/).
+
+## Specificity
+
+- Handle CSS specificity carefully to avoid conflicts.
+
+> Why? Specificity can cause styles to be overridden unintentionally. In CSS-in-JavaScript, the order in which style objects are applied matters, and higher specificity selectors can lead to unexpected results.
+
+```js
+// bad
+const styles = {
+  button: {
+    color: "red",
+  },
+  ".specialButton": {
+    color: "blue",
+  },
+};
+
+// good
+const styles = {
+  button: {
+    color: "red",
+  },
+  specialButton: {
```

Comment:
why is this an improvement? `.specialButton` targets a class, and `specialButton` targets a nonexistent tag name. did you mean `'button.specialButton'`?

Reply:
specialButton in the second is just a property to styles. It is not a class we pass it using styles.specialButton. It is a css-in-js syntax

Reply:
can you elaborate? I’m not familiar with any css-in-js solution that conflates tag names and class names in this way.